projects/create-and-vary-a-licence-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/service/LicenceConditionApplier.kt

```diff
@@ -65,13 +65,42 @@ class LicenceConditionApplier(
                 )
             }
 
-            else -> listOf(
-                ActionResult.Ignored(
-                    "Multiple Custodial Sentences",
-                    properties
-                )
+            else -> associateLicenceToLongestSentence(sentences, com, activatedLicence, occurredAt, properties)
+        }
+    }
+
+    fun associateLicenceToLongestSentence(
+        sentences: List<Custody>,
+        com: PersonManager,
+        activatedLicence: ActivatedLicence,
+        occurredAt: ZonedDateTime,
+        properties: Map<String, String>
+    ): List<ActionResult> {
+        val longestSentenceByEnteredDate = sentences
```

Comment:
Does longest mean the latest by end date here?  Or are we assuming the start date is the same on all matching sentences?

Reply:
i mean latest by end date.  i will rename the method to make it clearer.